canonical-websites/insights.ubuntu.com
# Update homepage banner

## Done

- Update homepage banner to normal size p-strip

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [homepage](http://127.0.0.1:8023/)
- See that the banner is not deep or shallow
